components/dropdown/basic/src/index.scss

```diff
@@ -100,36 +100,32 @@
       background-color: $bgc-dropdown-menu;
       border-top: $bdt-dropdown-menu;
       display: block;
-      padding: $gutter;
-      padding-bottom: $gutter-s;
+      padding: $gutter-s;
```

Comment:
Makes sense that it keeps setting a gutter value?

Reply:
As we've commented before, I've modified this shape variables to standard naming